src/new.rs

```diff
@@ -74,7 +72,7 @@ impl Creator {
             }
 
             Template::App => {
-                crate::fs::mkdir(&self.src.join(&self.options.name))?;
+                crate::fs::mkdir(&self.src.join(self.options.name.clone().unwrap()));
```

Comment:
In all these you can avoid cloning by doing `self.options.name.as_ref().unwrap()` instead

Reply:
And let's set a default in case the name was not specified. Calling unwrap will crash the program if there is no name. We can use the name of the target directory as a default and return and error if the directory name is not appropriate

Reply:
there is a comment from @lpil about avoid using `unwrap` I'll come back to this later, thanks for your suggestion.